artifactory/commands/transferconfig/transferconfig.go

```diff
@@ -353,6 +354,9 @@ func (tcc *TransferConfigCommand) waitForImportCompletion(targetServicesManager
 		return err
 	}
 	log.Info(fmt.Sprintf("Logs from Artifactory:\n%s", body))
+	if strings.Contains(string(body), "[ERROR]") {
+		return errorutils.CheckErrorf("Errors detected during config import. Hint: To skip failed repositories, please refer to the '--exclude-repos' flag.")
```

Comment:
Errors detected during config import. Hint: To skip failed repositories, please refer to the '--exclude-repos' flag.
-->
Errors detected during config import. Hint: You can skip transferring some Artifactory repositories by using the '--exclude-repos' command option. Run 'jf rt transfer-config -h' for more information.